java/restheart/pom.xml

```diff
@@ -9,7 +9,7 @@
   <properties>
     <maven.compiler.source>21</maven.compiler.source>
     <maven.compiler.target>21</maven.compiler.target>
-    <restheart.version>8.0.0-beta</restheart.version>
+    <restheart.version>[8.0,8.1.0-SNAPSHOT)</restheart.version>
```

Comment:
I think this is to guarantee to keep in 8.0.x

Side note :
Is there a way to use `[8.0,8.1)` and to exclude all releases containing SNAPSHOT string in its name. I know that some artefact on java world are named 4.1-RC but are 4.0 and not 4.1

Reply:
with `<restheart.version>[8.0,8.1)</restheart.version>` the version `8.1.0-SNAPSHOT` would be picked up. mvn is quite wierd with range version and this is the only way I found to exclude  8.1.0-SNAPSHOT

Reply:
Ok then, let's use this.

But you prefer to use gradle, be my guest. The package manager depends on the framework choice (I mean I prefer to use what developers of framework recommend)